.github/workflows/conformance-eks-v1.11.yaml

```
@@ -135,6 +135,16 @@ jobs:
           fi
 
           CILIUM_INSTALL_DEFAULTS="--cluster-name=${{ env.clusterName }} \
+            --chart-directory=install/kubernetes/cilium \
+            --helm-opts=image.repository=quay.io/cilium/cilium-ci \
+            --helm-opts=image.tag=${SHA} \
+            --helm-opts=operator.image.repository=quay.io/cilium/operator \
```

As above, for AWS CI I think this should be:
```suggestion
            --helm-opts=operator.image.repository=quay.io/cilium/operator-aws \
```

---

See https://github.com/cilium/cilium/pull/18898#discussion_r816948873.